packages/angular/cli/commands/new.md

```diff
@@ -0,0 +1,5 @@
+Creates and initializes a new Angular app that is the default project for a new workspace.
+
+* The new workspace folder is given the specified project name. The new app is created at the top level of the workspace folder.
+
+* The new app's configuration appears in the `projects` section of the `angular.json` workspace configuration file, under its project name.
```

Comment:
Please add a note somewhere about prompts. Something like this: The command prompts for information about features to include in the initial app project, such as routing and the CSS file to use.

Reply:
Actually I added this before, but messed up the PR and had to start over and recreate the text. Adding it back now.